src/transformers/onnx/features.py

```diff
@@ -79,7 +98,7 @@ def supported_features_mapping(
 
 
 class FeaturesManager:
-    if is_torch_available():
+    if is_torch_available() and not is_tf_available():
```

Comment:
There's a bit of duplicate logic in this module - perhaps the autoclass imports above should moved directly within `FeaturesManager`?